Examples/test-suite/javascript/using2_workers_runme.js

```diff
@@ -0,0 +1,11 @@
+try {
```

Comment:
Could add in a description of what this test is testing please. I'm afraid it isn't clear to me.